updates for zig 0.13.0

brandtcormorant opened the pull request:
Hello, these are just small updates for zig 0.13.0 compatibility.

brandtcormorant committed:
updates for zig 0.13.0

joncinque commented:
Thanks for your contribution! I just pushed up a conflicting change to .gitignore -- do you mind rebasing so we can get this in?

brandtcormorant committed:
update ci and build.zig.zon for 0.13.0

brandtcormorant commented:
Rebased! Also added a commit to update the main.yml and build.zig.zon for 0.13.0.

Sidenote: I'm getting started with the other zig solana stuff y'all have been working on and am happy to make more contributions

joncinque reviewed (approved):
Thanks! I'm currently maintaining these zig packages as a side project, so any community help is extremely appreciated
